pulse_actions/publisher.py

```diff
@@ -3,8 +3,12 @@
 
 It might become a real pulse publisher one day.
 """
+from __future__ import absolute_import
```

Comment:
I don't think you need this line. If I understand the documentation correctly, this just changes the default import behavior to assume absolute imports by default [1].

[1] https://docs.python.org/2/whatsnew/2.5.html#pep-328-absolute-and-relative-imports